.github/workflows/cd.yml

```diff
@@ -14,33 +18,35 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Install Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+
       - name: Install dependencies
         run: yarn
 
-      - name: Deploy storybooks and gh-pages
-        env:
-          DEPLOY_KEY: ${{secrets.ACTIONS_DEPLOY_KEY}}
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-        run: |
-          eval "$(ssh-agent -s)"
-          ssh-add - <<< "${DEPLOY_KEY}"
-          yarn deploy
-      - name: Set branch name
-        id: vars
-        run: echo ::set-output name=stage::${GITHUB_REF#refs/*/}
-      - name: Create badges
-        run: yarn run test:badges
-      - name: Commit badges
-        uses: EndBug/add-and-commit@v7
-        with:
-          author_name: Gitflow
-          author_email: git@wednesday.is
-          message: 'Update badges'
-          add: 'badges/'
-          push: false
-      - name: Pushing to a protected branch
-        uses: CasperWA/push-protected@v2
+      - name: Build
+        run: yarn build
+
+      - name: AWS S3 Deploy
+        uses: jakejarvis/s3-sync-action@v0.5.0
         with:
-          token: ${{ secrets.PUSH_TO_PROTECTED_BRANCH }}
-          branch: ${{ steps.vars.outputs.stage }}
-          unprotect_reviews: true
+          args: --acl public-read --follow-symlinks
+        env:
+          SOURCE_DIR: './build/'
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: ${{ secrets.AWS_REGION }}
+          PATHS: '/*'
```

Comment:
set the env at the top instead of setting it up seperately for deploy and invalidation

Reply:
okay I'll try that